templates/resources/fleet_package_policy.md.tmpl

```diff
@@ -0,0 +1,31 @@
+---
+subcategory: "Fleet"
+layout: ""
+page_title: "Elasticstack: elasticstack_fleet_package_policy Resource"
+description: |-
+  Creates or updates a Fleet Package Policy.
+---
+
+# Resource: elasticstack_fleet_package_policy
+
+Creates or updates a Fleet Package Policy.
+
+It is highly recommended that all inputs and streams are provided in the
+Terraform plan, even if some are disabled. Otherwise, differences may appear
+between what is in the plan versus what is returned by the Fleet API.
+
+The Kibana Fleet UI can be used as a reference for what data needs to be provided.
```

Comment:
Similarly it might be worth pointing back to the [docs](https://www.elastic.co/guide/en/fleet/8.10/add-integration-to-policy.html) here too.